src/Examples/Advanced/CustomUnitModels/methanol_param_VLE.py

```diff
@@ -103,6 +106,7 @@ def build(self):
              3: ["H2", ("Vap", "Liq")],
              4: ["CH3OH", ("Vap", "Liq")]}
 
+# Antoine coefficients assume pressure in mmHG and temperature in K
```

Comment:
This should be indented. It is a good idea to activate a linter in your IDE if you can, as this will highlight these types of issues for you.